README.md

```diff
@@ -87,6 +87,17 @@ INSERT INTO "schema_migrations" (version) VALUES
 ;
 ```
 
+By default the gem will remove some extensions that typically aren't needed in structure dumps. You can choose to keep all, or just some, of those extensions:
```

Comment:
I find this a little bit of a mental twist now, but it is consistent with what the current behavior is and with what was discussed.

My only suggested improvement might be if we can link to the extensions that are removed by default. That way a reader coming along later can review that in advance and is hopefully not surprised, or has a better chance of finding that vs. source diving.

Maybe hyperlink "will remove some extensions" and link to this line or something similar: https://github.com/lfittl/activerecord-clean-db-structure/pull/43/files#diff-ee618179a1cf9ab492c174c2d14f7f0fdb903d2a91076e08ab10676c2a285983R24

By the way, did you get a chance to test these changes? If you were able to run this unreleased version and try it with the default behavior, and customized behavior, that should be super awesome.

Reply:
Yeah I have been running this locally. It works great.

Added a link :)

I agree RE the double negatives, but I think that's needed for backward compat.